File: src/nomanssave/bH.java
package nomanssave;

class bH implements bK {
   // $FF: synthetic field
   final bE ey;

   bH(bE var1) {
      this.ey = var1;
   }

   public String getID() {
      return "TWordsLearnt";
   }

   public boolean isSpecial() {
      return true;
   }

   public String ab() {
      return Integer.toString(bE.a(this.ey).b(eU.ks));
   }

   public void l(String var1) {
      throw new RuntimeException("Cannot set words learnt");
   }
}

File: NMSSaveEditor/nomanssave/mixed/bH.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace NMSSaveEditor
{


public class bH : bK {
   public bE ey;

   public bH(bE var1) {
      this.ey = var1;
   }

   public string getID() {
      return "TWordsLearnt";
   }

   public bool isSpecial() {
      return true;
   }

   public string ab() {
      return Integer.toString(bE.a(this.ey).b(eU.ks));
   }

   public void l(string var1) {
      throw new Exception("Cannot set words learnt");
   }
}



}
